lib/foreman_inventory_upload/generators/metadata.rb

```diff
@@ -27,7 +27,9 @@ def render_report(metadata)
         @stream.object do
           @stream.simple_field('report_id', Foreman.uuid)
           @stream.simple_field('host_inventory_api_version', '1.0')
-          @stream.simple_field('source', 'Satellite')
```

Comment:
Let's leave the `source` field as is. It can indicate whether rh_cloud is generating this report, or other services.